docs/src/pages/[platform]/components/phonenumberfield/examples/PhoneNumberFieldThemeExample.tsx

```diff
@@ -12,7 +12,7 @@ const theme: Theme = {
   },
 };
 
-export const ThemingExample = () => (
+export const PhoneNumberFieldThemeExample = () => (
   <ThemeProvider theme={theme}>
     <PhoneNumberField label="Themed field" defaultCountryCode="+1" />
```

Comment:
I suggest we should hide the label. The color in dark mode is not contrasty.
![Screen Shot 2022-06-15 at 4 10 22 PM](https://user-images.githubusercontent.com/40295569/173957678-160d4d75-19cb-49e4-93b9-9ad742f3b5dc.png)